packages/web/src/components/tipping/tip-audio/SendTip.tsx

```diff
@@ -0,0 +1,237 @@
+import BN from 'bn.js'
+import cn from 'classnames'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { ReactComponent as IconQuestionCircle } from 'assets/img/iconQuestionCircle.svg'
+import IconNoTierBadge from 'assets/img/tokenBadgeNoTier.png'
+import { BadgeTier } from 'common/models/BadgeTier'
+import { SquareSizes } from 'common/models/ImageSizes'
+import { Supporter, Supporting } from 'common/models/Tipping'
+import { BNWei, StringAudio, StringWei } from 'common/models/Wallet'
+import { getAccountUser } from 'common/store/account/selectors'
+import { getProfileUser } from 'common/store/pages/profile/selectors'
+import { getSupporters, getSupporting } from 'common/store/tipping/selectors'
+import { sendTip } from 'common/store/tipping/slice'
+import { getAccountBalance } from 'common/store/wallet/selectors'
+import { getTierAndNumberForBalance } from 'common/store/wallet/utils'
+import {
+  formatWei,
+  parseAudioInputToWei,
+  stringWeiToBN,
+  weiToString
+} from 'common/utils/wallet'
+import Tooltip from 'components/tooltip/Tooltip'
+import UserBadges, { audioTierMapPng } from 'components/user-badges/UserBadges'
+import { useUserProfilePicture } from 'hooks/useUserProfilePicture'
+import ButtonWithArrow from 'pages/audio-rewards-page/components/ButtonWithArrow'
+
+import styles from './TipAudio.module.css'
+
+const messages = {
+  availableToSend: 'AVAILABLE TO SEND',
+  sendATip: 'Send Tip',
+  enterAnAmount: 'Enter an amount',
+  insufficientBalance: 'Insufficient Balance',
+  tooltip: '$AUDIO held in linked wallets cannot be used for tipping',
+  becomeTopSupporterPrefix: 'Tip ',
+  becomeTopSupporterSuffix: ' $AUDIO To Become Their Top Supporter'
+}
+
+export const SendTip = () => {
+  const dispatch = useDispatch()
+  const account = useSelector(getAccountUser)
+  const supportersMap = useSelector(getSupporters)
+  const supportingMap = useSelector(getSupporting)
+  const profile = useSelector(getProfileUser)
+  const profileImage = useUserProfilePicture(
+    profile?.user_id ?? null,
+    profile?._profile_picture_sizes ?? null,
+    SquareSizes.SIZE_150_BY_150
+  )
+
+  const accountBalance = (useSelector(getAccountBalance) ??
+    new BN('0')) as BNWei
+
+  const [tipAmount, setTipAmount] = useState<StringAudio>('' as StringAudio)
+  const [tipAmountBNWei, setTipAmountBNWei] = useState<BNWei>(
+    new BN('0') as BNWei
+  )
+
+  const { tier } = getTierAndNumberForBalance(weiToString(accountBalance))
+  const audioBadge = audioTierMapPng[tier as BadgeTier]
+
+  const [isDisabled, setIsDisabled] = useState(true)
+  const [hasError, setHasError] = useState(false)
+
+  const [
+    amountToTipToBecomeTopSupporter,
+    setAmountToTipToBecomeTopSupporter
+  ] = useState<BNWei | null>(null)
+  const [supporting, setSupporting] = useState<Supporting | null>(null)
+  const [topSupporter, setTopSupporter] = useState<Supporter | null>(null)
+
+  useEffect(() => {
```

Comment:
Would be good to comment these useEffects or pull them into their own descriptively named fns